NimbleMedium/Sources/Data/NetworkAPI/Models/Requests/UpdateCurrentUserRequest.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  UpdateCurrentUserRequest.swift
+//  NimbleMedium
+//
+//  Created by Minh Pham on 14/10/2021.
+//
+
+import Foundation
+
+struct UpdateCurrentUserRequest {
+    let username: String
```

Comment:
I prefer `Parameters` or `Data` instead of `Request`
```suggestion
struct UpdateCurrentUserParameters {

    let username: String
```

```suggestion
struct UpdateCurrentUserData {

    let username: String
```

Reply:
And how about creating this model in the `Domain` Layer instead?